webapp/components/navbar.jsx

```diff
@@ -504,7 +520,7 @@ export default class Navbar extends React.Component {
                             <a
                                 role='menuitem'
                                 href='#'
-                                onClick={() => this.setState({showEditChannelPurposeModal: true})}
+                                onClick={() => this.showChannelPurposeModal()}
```

Comment:
Can you change this to `onClick={this.showChannelPurposeModal}` to save creating an extra function each time this renders?